pkg/fixtures/purger_redis.go

```diff
@@ -28,7 +29,7 @@ func newRedisPurger(config cfg.Config, logger mon.Logger, name *string) (*redisP
 
 func (p *redisPurger) purge() error {
 	p.logger.Infof("flushing redis %s", *p.name)
-	_, err := p.client.FlushDB()
+	_, err := p.client.FlushDB(context.Background())
```

Comment:
Does it make sense to pass the application context also to the fixture loaders / purgers?

Reply:
If you look closer than you see that this is executed before there is an actual kernel => no context available

Reply:
That's not visible by only looking at your change set.